### DISCUSSION TITLE
The configure sso is not shown in my personal account.

### DISCUSSION THREAD

**Discussion by CurryXu** [community/community#122014] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I haven been added to an organization. The organization shared their repo to me. But when I generate a token, they need me to Configure SSO for that token, but it is my personal account, they Configure SSO is not shown in my personal account. I can't authenticate at least once through your SAML IdP to access resources on GitHub.com. Can someone help me to solve this? Or any other work around?

**Comment by pladhe001** [comment]
same issue

**Comment by ebndev** [comment]
Hey @CurryXu and @pladhe001!

Before I try to find some supporting docs on this, can you confirm:
- You've accepted the invitation to the organization
- You can see the organization in your settings under Organizations
- You've clicked into the org's repo (this should re-direct you to the SAML IdP login the first time)

**Reply by CurryXu** [reply]
Hi @ebndev  Thanks, we solved this to click the organization's SAML IdP link for us.

**Reply by ebndev** [reply]
@CurryXu that's good news! Would you mind sharing what you did to get it to work? That way I can mark it as an answer in case other folks have the same question in the future 😁

**Reply by CurryXu** [reply, marked as answer]
Hi @ebndev  Sure, actually the solution is still to ensure that you have authenticated at least once through your SAML IdP to access resources on GitHub.com.
Previously I can see all the repos in the organization and can clone/pull/push, but I thought my github account is my personal account, there is no way to do a Single Sign On since the email is not in the organization's AD group.
But the organization provide me an app which is GitHub Enterprise Cloud - Enterprise Account in the https://myapps.microsoft.com, then I click that app, it redirect the Single sign-on page to the organization.  Then I just click continue, it do the SSO for me with my personal email.
Hopefully that helps others.

**Reply by ebndev** [reply]
Awesome, thank you for sharing! @pladhe001 if you're still having the issue you can see if that works for you.
